server/service/handler.go

```diff
@@ -458,6 +458,9 @@ func attachFleetAPIRoutes(r *mux.Router, svc fleet.Service, config config.FleetC
 	ne.WithAltPaths("/api/v1/osquery/enroll").
 		POST("/api/osquery/enroll", enrollAgentEndpoint, enrollAgentRequest{})
 
+	ne.POST("/api/_version_/fleet/orbit/enroll", enrollOrbitEndpoint, enrollOrbitRequest{})
+	ne.POST("/api/_version_/fleet/orbit/flags", getOrbitFlagsEndpoint, orbitRequest{})
```

Comment:
```go
// unauthenticated endpoints - most of those are either login-related,
// invite-related or host-enrolling. So they typically do some kind of
// one-time authentication by verifying that a valid secret token is provided
// with the request.
ne := newNoAuthEndpointer(svc, opts, r, apiVersions...)
```

I can see that `"/api/_version_/fleet/orbit/flags"` already verifies the orbit node key itself, but we could introduce a:
```go
// orbit-authenticated endpoints
oe := newOrbitAuthenticatedEndpointer(svc, logger, opts, r, apiVersions...)
```
similar to osquery's one:
```go
// host-authenticated endpoints
he := newHostAuthenticatedEndpointer(svc, logger, opts, r, apiVersions...)
```

(Mostly to not rely on next endpoint we add to do the authentication too.)